src/main/resources/templates/home.html

```diff
@@ -0,0 +1,38 @@
+<html>
+<body>
+{{#name}}Hello, {{.}}!{{/name}}
+
+{{^name}}
+<form action="/login" method="post">
+    <input type="text" placeholder="Enter your name" name="userName"/>
+    <button type="submit">Login</button>
+</form>
+{{/name}}
+<br>
+
+<form action="/add-message" method="post">
+    <input type="text" placeholder="Enter Message" name="messageText"/>
+    <button type="submit">Submit</button>
+
+</form>
+
+<br>
+<ol>
+    {{#message}}
+    <li> <form action="/delete-message" method="post">
```

Comment:
This should be reorganized so that the add message form is not shown when the user is not logged in.